public/app/features/alerting/unified/mocks/server/handlers/mimirRuler.ts

```diff
@@ -23,6 +23,26 @@ export const updateRulerRuleNamespaceHandler = (options?: HandlerOptions) => {
   });
 };
 
+const rulerRulesHandler = () => {
+  return http.get<{ datasourceUid: string }>(`/api/ruler/:datasourceUid/api/v1/rules`, () => {
+    return HttpResponse.json({});
+  });
+};
+
+const rulerRulesTestHandler = () => {
+  /**
+   * This particular response is needed to allow tests to handle the case of testing
+   * whether a ruler supports rules (see `isCortexErrorResponse` method)
```

Comment:
Another fun heuristic piece of code that I had to unravel :D 
We are checking for a very particular part of the response message, so I added this in so the rule editor test could see "loki" in the dropdown